File: dichone-grounds/app/src/main/java/com/mafideju/saymyname/MainActivity.java
package com.mafideju.saymyname;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {
    private Button mainButton;
    private TextView mainTitle;
    private EditText mainField;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        mainButton = findViewById(R.id.mainButton);
        mainTitle = findViewById(R.id.mainTitle);
        mainField = findViewById(R.id.mainField);

        mainButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                String name = mainField.getText().toString();
                if (name.isEmpty()) {
                    mainTitle.setText("Java Android com nóis tudo!!.");
                } else {
                    mainTitle.setText("Java Android com " + name + ".");
                }
            }
        });
    }

    public Button getMainButton() {
        return mainButton;
    }

    public void setMainButton(Button mainButton) {
        this.mainButton = mainButton;
    }

    public TextView getMainTitle() {
        return mainTitle;
    }

    public void setMainTitle(TextView mainTitle) {
        this.mainTitle = mainTitle;
    }
}

File: mehmet-grounds/ToastsSnackbars/app/src/main/java/com/mafideju/toastssnackbars/MainActivity.kt
package com.mafideju.toastssnackbars

import android.content.DialogInterface
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Button
import android.widget.Toast
import androidx.appcompat.app.AlertDialog
import androidx.constraintlayout.widget.ConstraintLayout
import com.google.android.material.snackbar.Snackbar
import com.google.android.material.snackbar.SnackbarContentLayout

class MainActivity : AppCompatActivity() {
    lateinit var showToast: Button
    lateinit var buttonsnack: Button
    lateinit var layout: ConstraintLayout
    lateinit var buttonAlert: Button

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        showToast = findViewById(R.id.button)
        buttonsnack = findViewById(R.id.buttonsnack)
        layout = findViewById(R.id.layout)
        buttonAlert = findViewById(R.id.buttonAlert)

        showToast.setOnClickListener {
            Toast.makeText(applicationContext, "Este é um Toastie!", Toast.LENGTH_LONG).show()
        }
        buttonsnack.setOnClickListener {
            Snackbar.make(layout, "Este é um SnackBar Snickers", Snackbar.LENGTH_INDEFINITE)
                .setAction("Close", View.OnClickListener {  })
                .show()
        }
        buttonAlert.setOnClickListener {
            AlertDialog.Builder(this@MainActivity)
                .setTitle("Exemplo!")
                .setMessage("Este é um exemplo de AlertBox no Android")
                .setIcon(R.drawable.a24)
                .setCancelable(false)
                .setNegativeButton("Não", DialogInterface.OnClickListener { dialog, which ->
                    dialog.cancel()
                })
                .setPositiveButton("Sim", DialogInterface.OnClickListener { dialog, which ->
                    buttonAlert.text = "Clicado!"
                })
                .create()
                .show()
        }
    }
}